src/main/kubeconfig-manager.ts

```diff
@@ -24,14 +24,24 @@ export class KubeconfigManager {
   protected async init() {
     try {
       await this.contextHandler.ensurePort();
-      await this.createProxyKubeconfig();
+      this.tempFile = this.createProxyKubeconfig();
     } catch (err) {
       logger.error(`Failed to created temp config for auth-proxy`, { err });
     }
   }
 
   getPath() {
```

Comment:
This will return `""` if `this.tempFile` is `""`. Is that what we want?

Reply:
when `this.tempFile` will get ""?

Reply:
I don't know but your `if` check checks that `tempFile` is truthy (so not "") before checking if the file exists. If `tempFile` is falsey then it just returns that.

Reply:
`tempFile` should contain path to proxy kubeconfig, so I think we want to re-create it if that's `""`.

Reply:
exactly